packages/core/src/Renderer.ts

```diff
@@ -1,11 +1,12 @@
+import { VNode } from './VNodes/VNode';
 import { VDocument } from './VDocument';
 
 export type RendererIdentifier = string;
-export type RenderingFunction<Context = {}, T = {}> = (context: Context) => T;
+export type RenderingFunction<To = {}> = (node: VNode, to: To) => Map<VNode, To[]>;
```

Comment:
That is kinda weird... If you want to render a node into string you have to pass it a string ?